vars/deployToCluster.groovy

```diff
@@ -2,7 +2,7 @@ def call(Map config=[:], Closure body={}) {
   node {
     try {
       stage('Deploy Helm chart') {
-        helm.deployRemoteChart(config.environment, config.namespace, config.chartName, config.chartVersion)
+        helm.deployRemoteChart(config.environment, config.namespace, config.chartName, config.chartVersion, config.helmChartRepoType)
```

Comment:
Add the following before running deployRemoteChart: `deleteDir()`
If the helm deploy fails it leaves the helm-install dir which might include an older verison of the helm chart library that will get picked up instead of the new one (I know this from bitter experience)

Reply:
I've added `deleteDir()` but not to `deployToCluster`, I've added it to `vars/helm` so should the function be used to construct a bespoke pipeline they will benefit from the deletion of the dir.

Reply:
As discussed, I've moved the deletion of the dir into the `Helm.groovy` class to clear up helm chart directories.